addons/cargo/functions/fnc_setSpace.sqf

```diff
@@ -31,14 +31,22 @@ TRACE_2("setSpace",_vehicle,_size);
 // Nothing to do here
 if (
     (isNil "_space") ||
-    {isNull _vehicle} ||
-    {_space == _vehicle getVariable [QGVAR(space), CARGO_SPACE(typeOf _vehicle)]}
+    {isNull _vehicle}
 ) exitWith {};
 
 // Apply new space globally
```

Comment:
maybe this comment should belong to `setVariable` lines below

Reply:
Rearranged the comments and added a couple of newlines for readability